src/solvers/refinement/bv_refinement.h

```diff
@@ -30,6 +31,10 @@ class bv_refinementt:public bv_pointerst
     bool refine_arrays=true;
     /// Enable arithmetic refinement
     bool refine_arithmetic=true;
+    /// Random value for the SAT solver
+    double random_seed=satcheck_minisat2_baset::default_random_seed;
+    /// Random value frequency for the SAT solver
+    double random_var_freq=satcheck_minisat2_baset::default_random_var_freq;
```

Comment:
Why are these being assigned here?  This seems a weird place to be setting up default values.  Either at the top (where the options are parsed) or at the bottom (where they are used) seem sensible.